docs/blog/2019-06-10-how-to-recognize-when-gatsby-is-a-good-fit-for-your-client/index.md

```diff
@@ -39,7 +39,7 @@ Gatsby is a bleeding-edge approach to solving problems, and it’s pretty differ
 
 ## #3 They understand site speed is more than page load times
 
-Mobile websites that take [more than 3 seconds to load have a 53% bounce rate](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/), meaning that you lose more than half of your visitors because they would rather try to restart their search than continue waiting for your site. In the ecommerce domain, some estimates say you lose up to [1% of revenue for every 100ms delay](https://www.section.io/blog/page-load-time-bounce-rate/) in page load time. When performant technologies like Gatsby become more common, those numbers are going to become more severe.
+Mobile websites that take [more than 3 seconds to load have a 53% bounce rate](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/), meaning that you lose more than half of your visitors because they would rather try to restart their search than continue waiting for your site. In the E-commerce domain, some estimates say you lose up to [1% of revenue for every 100ms delay](https://www.section.io/blog/page-load-time-bounce-rate/) in page load time. When performant technologies like Gatsby become more common, those numbers are going to become more severe.
```

Comment:
```suggestion
Mobile websites that take [more than 3 seconds to load have a 53% bounce rate](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/), meaning that you lose more than half of your visitors because they would rather try to restart their search than continue waiting for your site. In the e-commerce domain, some estimates say you lose up to [1% of revenue for every 100ms delay](https://www.section.io/blog/page-load-time-bounce-rate/) in page load time. When performant technologies like Gatsby become more common, those numbers are going to become more severe.
```